### ISSUE TITLE
Failure with Python 3.5

### ISSUE THREAD

**Issue by orbeckst** [jensengroup/propka-3.1#63]
While trying tor reproduce #48 I ran into a different issue, possibly related to 3.5. I am checking now and will update the issue as I go along. (Possibly the new CI will also show problems #62).

However, 3.5 is not officially supported anymore so we could easily drop it from PROPKA 3.2.0, just needs changes in PR #61 

## Reproduce with
```bash
# I use a virtualenv
mkvirtualenv -p python3.5 propka    # use a virtualenv (old system with only py3.5)
pip install ipython numpy                  # not needed but useful for testing
git clone https://github.com/jensengroup/propka-3.1.git
pip install -v propka-3.1/
```
I will then test with
```
cd /tmp
curl https://files.rcsb.org/download//2bem.pdb.gz | gzip -dc > 2bem.pdb
propka31 2bem.pdb
```
... and I get the following error
```
Traceback (most recent call last):
  File "~/.virtualenvs/propka/lib/python3.5/site-packages/propka/input.py", line 27, in open_file_for_reading
    file_ = open(input_file, 'rt')
TypeError: invalid file: PosixPath('2bem.pdb')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "~/.virtualenvs/propka/bin/propka31", line 8, in <module>
    sys.exit(main())
  File "~/.virtualenvs/propka/lib/python3.5/site-packages/propka/run.py", line 21, in main
    my_molecule = read_molecule_file(pdbfile, my_molecule)
  File "~/.virtualenvs/propka/lib/python3.5/site-packages/propka/input.py", line 51, in read_molecule_file
    input_path, mol_container.version.parameters, mol_container)
  File "~/.virtualenvs/propka/lib/python3.5/site-packages/propka/input.py", line 288, in read_pdb
    for (name, atom) in lines:
  File "~/.virtualenvs/propka/lib/python3.5/site-packages/propka/input.py", line 127, in get_atom_lines_from_pdb
    lines = open_file_for_reading(pdb_file).readlines()
  File "~/.virtualenvs/propka/lib/python3.5/site-packages/propka/input.py", line 29, in open_file_for_reading
    raise IOError('Cannot find file {0:s}'.format(input_file))
TypeError: non-empty format string passed to object.__format__
```

**Comment by orbeckst** [comment]
According to the [Py 3.6 release notes](https://docs.python.org/3/whatsnew/3.6.html#pep-519-adding-a-file-system-path-protocol), PathLib as input of `open()` is new in 3.6.

My recommendation is to drop 3.5 support for this release.

**Comment by sobolevnrm** [comment]
Fixed in 11bc32e9fcea71bec87681aab87ff8d52dde49b1
